yt/config.py

```diff
@@ -190,6 +217,31 @@ def get_global_config_file():
     def get_local_config_file():
         return os.path.join(os.path.abspath(os.curdir), "yt.toml")
 
+    def _convert_legacy_logging_parameters(self):
+        # YTEP-0039: maintain backward compatibility for a period of time
+        # after which this function should be removed.
+        try:
+            self["logging", "use_color"] = self.get("yt", "colored_logs")
```

Comment:
For consistency I'd favour
```suggestion
            self["logging", "use_color"] = self["yt", "colored_logs"]
```

Reply:
I wonder: did you keep the `get` method just for backwards compatibility ? In which case I'm going to drop it everywhere this is currently used in lines touched by this PR (and soon, the world)